portfolio/src/main/java/com/google/sps/data/Image.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
```

Comment:
I'm guessing that this file (and perhaps Marker.java and BlobstoreHandlerServlet.java is the template code? But the github diff is showing it all as new code. If it's not your code, you can ignore the comments.